<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_product_menu</class>
 <widget class="QWidget" name="page_product_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_to_idle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_help_page">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>1732</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_menu_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>188</y>
     <width>1080</width>
     <height>86</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_help">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>1773</y>
     <width>125</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_base_products_section_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>270</width>
     <height>1600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_base_products_section_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>345</y>
     <width>270</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_base_product_bg_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>170</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_base_product_bg_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>671</y>
     <width>170</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_base_product_bg_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>892</y>
     <width>170</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_base_product_bg_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>1113</y>
     <width>170</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_product_menu_title</zorder>
  <zorder>label_help</zorder>
  <zorder>pushButton_help_page</zorder>
  <zorder>pushButton_to_idle</zorder>
  <zorder>label_base_products_section_bg</zorder>
  <zorder>label_base_products_section_title</zorder>
  <zorder>label_base_product_bg_1</zorder>
  <zorder>label_base_product_bg_2</zorder>
  <zorder>label_base_product_bg_3</zorder>
  <zorder>label_base_product_bg_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
